### PR TITLE
support multi txns testing

### PR DESCRIPTION
**Pull request by yuunlimm**

### CODE REVIEW

**Comment by yuunlimm** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/555?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#555** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/555?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#554** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/554?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#553** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-indexer-processors/553?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @yuunlimm and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: rust/integration-tests/sdk_expected_db_output_files/imported_testnet_txns/fungible_asset_processor/278556781/coin_supply.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
     "coin_type": "0x1::aptos_coin::AptosCoin",
     "supply": "18544645433533405815",
     "transaction_timestamp": "2022-10-13T01:46:57",
-    "transaction_epoch": 746,
-    "inserted_at": "2024-10-15T05:16:56.913064"
```
Comment (dermanyang) [inline rust/integration-tests/sdk_expected_db_output_files/imported_testnet_txns/fungible_asset_processor/278556781/coin_supply.json:9]: weird. I thought these `inserted_at` fields were removed in a previous PR already

**Review by dermanyang — APPROVED** [review]

File: rust/integration-tests/src/sdk_tests/events_processor_tests.rs [hunk 2]
```diff
@@ -1,28 +1,64 @@
...
+use std::collections::HashSet;
+
+pub fn setup_events_processor_config(
+    test_context: &SdkTestContext,
+    txn_version: u64,
+    txn_count: usize,
+    db_url: &str,
+) -> (IndexerProcessorConfig, &'static str) {
+    let transaction_stream_config =
+        test_context.create_transaction_stream_config(txn_version, txn_count as u64);
```
Comment (yuunlimm) [inline rust/integration-tests/src/sdk_tests/events_processor_tests.rs:17]: @rtso

File: rust/integration-tests/src/sdk_tests/events_processor_tests.rs [hunk 3]
```diff
@@ -98,4 +116,54 @@ mod tests {
             }
         }
     }
+
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+    async fn testnet_events_processor_db_output_scenario_testing() {
```
Comment (yuunlimm) [inline rust/integration-tests/src/sdk_tests/events_processor_tests.rs:121]: @rtso

File: rust/integration-tests/sdk_expected_db_output_files/imported_testnet_txns/fungible_asset_processor/278556781/coin_supply.json (hunk 1 above)
Comment (yuunlimm) [reply rust/integration-tests/sdk_expected_db_output_files/imported_testnet_txns/fungible_asset_processor/278556781/coin_supply.json:9]: this rebasing is tricky.. they are removed in the next stacked pr.

**Pull request merged**